File: C program/year.c
#include<stdio.h>
int main()
{
	int r,year;
	
	printf("Enter the year:\n");
	scanf("%d",&year);
	
	r = year%4;
	
	if(r==0)
	printf("Entered year is a leap year.");
	else
	printf("Entered year is not a leap year.");
}


/*
----------output----------
Enter the year:
2024
Entered year is a leap year.
*/

File: Java Programming/year.java
public class year
{
    public static void main(String[] args)
    {
        int year = 2024;
        boolean isleapyear = false;
        if(year%4==0)
        {
            isleapyear=true;
            if(isleapyear)
            {
                System.out.println(year + " is a leap year.");
            }
        }
        else
        {
            System.out.println(year + " is not a leap year.");
        }
    }
}





/* 
--------------------output--------------------
2024 is a leap year.
*/